Source/Includes/Wordlist.ahk

```diff
@@ -467,12 +467,43 @@ StrUnmark(string) {
    ; Remove combining marks and return result.
    string := RegExReplace(StrGet(&buf, len, "UTF-16"), "\pM")
    
-   StringReplace, string, string, �, ae, All
-   StringReplace, string, string, �, AE, All
-   StringReplace, string, string, �, oe, All
-   StringReplace, string, string, �, OE, All
-   StringReplace, string, string, �, ss, All   
+   StringReplace, string, string, æ, ae, All
+   StringReplace, string, string, Æ, AE, All
+   StringReplace, string, string, , oe, All
+   StringReplace, string, string, , OE, All
+   StringReplace, string, string, ß, ss, All   
```

Comment:
Can you please revert the changes to these lines? If not, let me know and I'll take care of it.